client/src/reducers/recipeReducer.js

```diff
@@ -2,12 +2,12 @@ import {
   CREATE_RECIPE_SUCCESS, CREATE_RECIPE_FAILURE,
   GET_RECIPES,
   DELETE_RECIPE_SUCCESS,
-  EDIT_RECIPE_SUCCESS, SEARCH_RECIPE_SUCCESS, SEARCH_RECIPE_FAILURE, GET_PAGE_DETAIL, GET_UPVOTED_RECIPES_SUCCESS, GET_ALL_RECIPES_FAILURE
+  EDIT_RECIPE_SUCCESS, SEARCH_RECIPE_SUCCESS, SEARCH_RECIPE_FAILURE, GET_PAGE_DETAIL, GET_UPVOTED_RECIPES_SUCCESS, GET_MOST_FAVORITED_RECIPES
 } from '../actions/types';
 
 /* eslint-disable */
```

Comment:
What are you scared of?
Lol

Wondering why you have an `eslint-disable` in almost every file. What's the purpose of having `ESLint` if you're gonna disable it. 
Don't be scared of linting lol.